fairmq/zeromq/FairMQMessageZMQ.cxx

```diff
@@ -12,60 +12,71 @@
  * @author D. Klein, A. Rybalchenko, N. Winckler
  */
 
-#include <cstring>
-#include <cstdlib>
 
 #include "FairMQMessageZMQ.h"
 #include "FairMQLogger.h"
+#include "fairmq/Tools.h"
```

Comment:
I would prefer `#include <fairmq/Tools.h>` here. `""` mean here, look first in `fairmq/zeromq/fairmq/Tools.h`.

Or use `#include "../Tools.h"`. This should not be done in header files though.

Reply:
fixed.